spec/models/user_spec.rb

```diff
@@ -119,4 +120,27 @@
 
     expect(user.safe_user_name).to eq('aNTs-fully-s-ck-user-name-1337')
   end
+
+  it 'defaults to contactable: unasked' do
+    user = build(:user)
+    expect(user.contactable).to eq('unasked')
+    expect(user).to be_valid
+  end
+
+  it 'allows valid contactable values' do
+    expect(build(:user, contactable: 'consented')).to be_valid
+    expect(build(:user, contactable: 'unconsented')).to be_valid
+  end
+
+  it 'does not allow invalid contactable values' do
+    user = build(:user, contactable: 'donkey')
+    expect(user).not_to be_valid
+  end
+
+  it 'contactable status can be set and checked using scopes' do
+    user = build(:user)
+    user.contactable_consented!
+    expect(user.contactable_consented?).to be true
+    expect(user).to be_valid
```

Comment:
technically you didn't use a scope here. That would be more like:

```
User.contactable_consented.all
```

Which would load all users who had consented to contact. A scope is like a pre-made query for the database.

Either change the name of the test or actually test a scope.